src/EasyCronJob.AutoConfigurer/Startup.cs

```diff
@@ -0,0 +1,148 @@
+﻿using EasyCronJob.Abstractions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyCronJob.AutoConfigurer
+{
+    /// <summary>
+    /// This class includes Auto Configure for Cron Services base methods
+    /// </summary>
+    public static class Startup
+    {
+        /// <summary>
+        /// Find Cron Job Services
+        /// </summary>
+        /// <returns>
+        /// Returns <see cref="List{Type}"/>
+        /// </returns>
+        private static List<Type> FindCronJobServices()
+        {
+            var baseCronJobType = typeof(CronJobService);
+            var cronJobServices = AppDomain.CurrentDomain.GetAssemblies().SelectMany(s => s.GetTypes())
```

Comment:
I recommend to use `Assembly.GetEntryAssembly().GetReferencedAssemblies()` here.

This'll ignore 2nd level references. So, unwanted registrations can be prevented.

Reply:
Thanks for the suggestion.
Could you give some more detailed information about the risks?
@enisn

Reply:
In an example, I'm using EasyCronJob in my library and I have a couple of CronJobService, but I only register only one of them. Or I'm configuring them according to a condition, my code picks one of my CronJobService and registers it according to a condition. 

If your application has that autoconfiguration, and depends on my library, you'll register all CronJobServices in my library and all app domain assemblies too, even if I don't want to register all of them.